src/DateInput.js

```diff
@@ -49,7 +71,10 @@ export default class DateInput extends Component {
         delete props.onBlur;
 
         return (
-            <Input
+            <MaskedInput
+                mask={[/\d/, /\d/, '-', /\d/, /\d/, '-', /\d/, /\d/,  /\d/, /\d/]}
```

Comment:
Same as mentioned above about the hardcoded format.

Reply:
I am going to remove that, cuz it is actually not necessary with the format in the datePipe included.